sched/scheduler/task_scheduler.go

```diff
@@ -33,6 +36,87 @@ func getTaskAssignments(cs *clusterState, tasks []*taskState) ([]taskAssignment,
 		snapshotIds = append(snapshotIds, snapId)
 	}
 
+	// Sort jobs by priority.
```

Comment:
I'm going to punt on that since it's unlikely we'll have time for a different approach for now.